### PR TITLE
update faqs

### PR DESCRIPTION
**Pull request by ilariae**
### Description

Updated faqs with the last questions that were added in the integrator faq google sheet https://docs.google.com/spreadsheets/d/1liKj0aT0EH6eIGX7bIVPDUkcCzemMyok48J7heylBlA/edit?gid=0#gid=0 

### Checklist

- [x] **Required** - I have added a label to this PR 🏷️
- [x] **Required** - I have run my changes through Grammarly
- [ ] If pages have been moved, I have created redirects in the `wormhole-mkdocs` repo

### CODE REVIEW

**Review by martin0995 — APPROVED** [review]
LGTM

**Review by albertov19 — APPROVED** [review]
LGTM

File: build/contract-integrations/native-token-transfers/faqs.md [hunk 1]
```diff
@@ -117,3 +117,12 @@ Shortcomings of xERC20:
 - **No integration with relaying systems** - xERC20 does not natively support relayer systems, limiting its usability in automated or dynamic setups
 
 While xERC20 is an extension of the ERC20 standard, NTT is designed as a framework rather than a rigid standard. It is compatible with any token that supports `burn` and `mint` functions and allows the NTT manager to act as a minter. 
+
+## Can I set `IsSpecializedRelayer` to true via the CLI?
+
+Not currently. However, we are working on enabling this functionality. Once the latest CLI version is released and you have upgraded, you will be able to set `IsSpecializedRelayer` to `true` via the CLI.
```
Comment (eshaben) [inline build/contract-integrations/native-token-transfers/faqs.md:123]: Just happened to see this now. Are we keeping track of this @ilariae ?

Comment (ilariae) [reply build/contract-integrations/native-token-transfers/faqs.md:123]: good point, i just asked about it @eshaben

**Pull request merged**